patroni.spec

```diff
@@ -16,7 +16,7 @@ def hiddenimports():
 a = Analysis(['patroni/__main__.py'],
              pathex=[],
              binaries=None,
-             datas=None,
+             datas=[('patroni/postgresql/available_parameters', 'patroni/postgresql/available_parameters')],
```

Comment:
Works for me too! At a first moment I just added the full directory because the logic in `validator.py` would filter anything other than YAML files out.
But I also think it's better if we make it more explicit about the expected content of the directory in the spec/setup files.
Thanks for the pointer on the pip packaging btw.